core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/EntityApiService.scala

```diff
@@ -202,14 +205,18 @@ trait EntityApiService extends UserInfoDirectives {
                                            billingProject
                       )
                       .map(_ => StatusCodes.NoContent)
+                      .recover { case _: EntityStreamSizeException =>
+                        StatusCodes.PayloadTooLarge
+                      }
```

Comment:
switching to streams changes how `withSizeLimit` throws errors. This `recover` preserves the previous behavior of returning a 413 status code.